vnext/Desktop.DLL/react-native-win32.x64.def

```diff
@@ -50,8 +50,10 @@ EXPORTS
 ?typeName@dynamic@folly@@QEBAPEBDXZ
 ?updateProperties@ShadowNode@react@facebook@@UEAAX$$QEBUdynamic@folly@@@Z
 ?Utf16ToUtf8@Unicode@Common@Microsoft@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@AEBV?$basic_string@_WU?$char_traits@_W@std@@V?$allocator@_W@2@@5@@Z
-?SetFeatureGate@React@Microsoft@@YAX$$QEAV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@_N@Z
```

Comment:
If we're changing name, we're likely not going to want to backport to 0.60 since it would break existing usages in older Office forks that we want to be able to rev if needed. In general we've been wanting to avoid backporting anything breaking.

Consider keeping the API as is, or scoping backports to 0.62/0.63. I think the latter would probably be less of a hassle if the customer need this is addressing can wait a month.

Reply:
I took this into account.

We recently came down from 1 to 0 usages of this API in Office.

The change won’t break any existing code.

Reply:
Is it unused in past forks as well? I'm less concerned about the current fork and more about the scenario of we see a security concern pop up and need to rev old versions of 0.60 in other forks. Tudor had a painful time changing RN version in a fork due to a similar API rename before. There's an old email thread we had called "Office forks in RNW" last April with some context from Tudor and ACoates running into issues with the WebSocketResource rename. 

For the public RNW interface we've been getting pretty strict about not changing APIs for released versions at all unless the changes are additive. I think we've been wanting to do that for Office APIs as well, but @khalef1 will have more context on how concrete that is vs aspirational 🙂.

Reply:
It got added after June’s fork and removed before July fork.

It only had one usage.